src/js/other-websites/inpage_menu.js

```diff
@@ -193,26 +276,64 @@ function applySearchFilter(query) {
     }
   });
 
-  // Add #/# labels to search filter
+  // Set #/# labels inside search bar to show results count
   const searchFilterTotal = document.querySelector(".js-filter-masks-total");
   const searchFilterVisible = document.querySelector(
     ".js-filter-masks-visible"
   );
 
-  searchFilterVisible.textContent = maskResults.length;
+  searchFilterVisible.textContent = maskSearchResults.filter((maskResult) => !maskResult.classList.contains("is-hidden")).length;
+  searchFilterTotal.textContent = maskSearchResults.length;
+}
+
+function checkIfAnyMasksWereGeneratedOnCurrentWebsite(masks, domain) {
+  return masks.some((mask) => {
+    return domain === mask.generated_for;
+  });
+}
```

Comment:
praise: From all of us less fluent in JS, TY for making this a nicely-named function instead of a terse, dense, and cryptic 1-liner. 😄